### PR TITLE
Sammyak

### PR DESCRIPTION
**Pull request by SammyakFlair**
1. Base url changed

### CODE REVIEW

**Comment by SammyakFlair** [comment]
@punit-fm I cant see any comments

**Review by punit-fm — CHANGES_REQUESTED** [review]

File: src/components/BackTestNavbar/BackTestNavbar.jsx [hunk 1]
```diff
@@ -41,11 +41,15 @@ export default function BackTestNavbar({codeValue, showDrawer}) {
 		}
 	};
 
-	const saveToFile = () => {
-		const blob = new Blob([codeValue], { type: "text/plain" });
+	const saveToFile = async() => {
```
[the remark this answers is not in the document]
Comment (SammyakFlair) [reply src/components/BackTestNavbar/BackTestNavbar.jsx:44]: error handling is done

File: src/layout/SubSidebar/ChartSubSidebar/ChartCanvas.jsx [hunk 2]
```diff
@@ -84,7 +85,7 @@ const ChartCanvas = ({ setCodeValue }) => {
 			setIsLoading(true);
 	
 			const payload = {
-				code: JSON.stringify(codeSnippetData)
+				code: (codeSnippetData)
 			};
 	
 			const response = await saveScriptData(graphName, payload);
```
[the remark this answers is not in the document]
Comment (SammyakFlair) [reply src/layout/SubSidebar/ChartSubSidebar/ChartCanvas.jsx:91]: Error handling is done

File: src/pages/PlumDashboard/PlumDashboard.jsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ import ChatDrawer from '../../components/ChatDrawer/ChatDrawer';
 export default function PlumDashboard() {
 	const [codeValue, setCodeValue] = useState(null);
 	const [open, setOpen] = useState(false);
+	const [graphNm, setGraphNm] = useState()
```
[the remark this answers is not in the document]
Comment (SammyakFlair) [reply src/pages/PlumDashboard/PlumDashboard.jsx:14]: Added empty string for this

**Pull request merged**